lib/Parser/RegexCompileTime.cpp

```diff
@@ -2659,7 +2680,7 @@ namespace UnifiedRegex
                     //     L3:    <item3>
```

Comment:
Note: you may want to update this comment; it's still referring to 10|20

Reply:
Thanks.